pkg/internal/sqlprune/oteljavasql_test.go

```diff
@@ -1,181 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-package sqlprune
-
-var otelJavaSQLQueries = []string{
-	"SELECT * FROM TABLE WHERE FIELD=1234", "SELECT * FROM TABLE WHERE FIELD=?",
-	"SELECT * FROM TABLE WHERE FIELD = 1234", "SELECT * FROM TABLE WHERE FIELD = ?",
```

Comment:
I merged this file with the actual test file and added validation of the output.